lib/restbase/rest.go

```diff
@@ -52,6 +54,15 @@ func Wrap(fn func(context.Context, *http.Request) (interface{}, error)) func(htt
 
 		res, err := fn(ctx, r)
 		if err != nil {
+			// rewrite common errors to internal error standard
+			if err == io.EOF {
+				err = cher.New(cher.EOF, nil)
+			} else if err == io.ErrUnexpectedEOF {
+				err = cher.New(cher.UnexpectedEOF, nil)
+			} else if strings.Contains(err.Error(), context.Canceled.Error()) {
```

Comment:
I definitely feel like we should find a better way of checking for this. Would have just thought `errors.Is(err, context.Canceled)` would be better... but maybe best to consider separately/later to keep it consistent with our internal codebase for now